clients/client/src/main/java/org/projectnessie/client/api/NessieApiV1.java

```diff
@@ -21,7 +21,7 @@
 import org.projectnessie.model.LogResponse;
 import org.projectnessie.model.NessieConfiguration;
 
-/** Interface for the Nessie {@link NessieApiVersion#V_1} API implementation. */
```

Comment:
Why all these javadoc changes? The enum's still there.

Reply:
I forgot to delete the enum